include/rangers-api/ApplicationCommon/FSM/GOCTinyFsm2.h

```diff
@@ -0,0 +1,96 @@
+#pragma once
+
+namespace app_cmn::fsm {
+    template<typename T>
+    class GOCTinyFsm2;
+
+    template<>
+    class GOCTinyFsm2<hh::game::GameObject> : public hh::game::GOComponent {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.